includes/abstracts/llms-abstract-admin-wizard.php

```diff
@@ -0,0 +1,390 @@
+<?php
+/**
+ * Display a Wizard
+ *
+ * @package LifterLMS/Admin/Classes
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Display a Wizard class
+ *
+ * @since [version]
+ */
+abstract class LLMS_Abstract_Admin_Wizard {
+
+	/**
+	 * Wizard type.
+	 *
+	 * @var string
+	 */
+	protected string $type = 'setup';
+
+	/**
+	 * Views directory
+	 *
+	 * @var string
+	 */
+	protected string $views_dir = LLMS_PLUGIN_DIR . 'includes/admin/views/setup-wizard/';
+
+	/**
+	 * Steps
+	 *
+	 * @since [version]
+	 *
+	 * @var array
```

Comment:
```suggestion
	 * @since [version]
	 * @var array
```
https://github.com/gocodebox/lifterlms/blob/trunk/docs/documentation-standards.md#class-members